rector.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\CodeQuality\Rector\Class_\InlineConstructorDefaultToPropertyRector;
+use Rector\Config\RectorConfig;
+use Rector\Set\ValueObject\LevelSetList;
+use Sylius\SyliusRector\Set\SyliusPlus;
```

Comment:
Are these imports necessary? Other than `RectorConfig`, of course.

Reply:
I'll check it 🚀